test/support/api/flows.js

```diff
@@ -45,7 +45,7 @@ function generateData(patients = _.sample(this.fxPatients, 1)) {
     };
 
     if (_.random(1)) {
-      const clinician = _.sample(this.fxClinicians);
+      const clinician = _.sample(_.rest(this.fxClinicians));
```

Comment:
Exclude the `Clinician McTester` (`id: '11111'`) from being auto-assigned as the owner to a flow. This was causing some flakiness with the find in list test specifically.

@paulfalgout should this being added to the `routeActions` command too ([here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/support/api/actions.js#L34))?